docs/_tutorials/local_development.md

```diff
@@ -12,55 +12,43 @@ Feel free to skip around to the specific sections relevant to you — we won't m
-
-You can learn more about the different token types [on our API site](https://api.slack.com/docs/token-types). The type of token your app needs depends on the actions you want it to perform. But for brevity, we're going to use bot tokens for this guide.
-
-To add a bot user, click **Bot Users** on the left sidebar and then **Add A Bot User**. Give it a display name and username and then click **Add Bot User**.
+Head over to our [Bolt JS Getting Started Guide's section "Tokens and installing apps"](https://slack.dev/bolt-js/tutorial/getting-started#tokens-and-installing-apps) for the latest up-to-date information on what tokens are, the [different kinds of tokens available](https://api.slack.com/docs/token-types), how to create them on [api.slack.com/apps](https://api.slack.com/apps) and finally how to install your app to a live Slack workspace.
 
-Now that you have a bot user with permission to send messages to Slack, let's install the app to your workspace.
+## Socket Mode vs. HTTP
+Your app can [communicate with Slack using one of two methods](https://api.slack.com/apis/connections):
 
-Click **Install App** on the left sidebar and click the big **Install App to Workspace** button at the top of the page. Once you authorize the installation, you'll land on the **OAuth & Permissions** page.
+1. [Socket Mode](https://api.slack.com/apis/connections/socket). Connect to Slack using a direct and long-lived socket connection. While convenient, easy to setup and the recommended approach when getting started or iterating on your app, apps cannot use Socket Mode and be listed in the App Directory (for that you'll need to use [HTTP](#http)). It is also important to remember that Socket Mode is prone to network faults because the connection is long-lived.
+2. [HTTP](https://api.slack.com/apis/connections/events-api). Expose your app using a public-facing URL that Slack will send HTTP POST requests to. A bit harder to set up, but more resilient to network disruptions.
 
-![OAuth Tokens](bot-token.png "OAuth Tokens")
+This tutorial will cover how to set up your app using either approach.
 
-You'll see two tokens. To add scopes to the `xoxp` tokens, you can scroll down to the **Scopes** section. For now, we'll just use the `xoxb` bot token.
+### Socket Mode
 
-You'll need your token to use any of the [Web API methods](https://api.slack.com/methods).
+Setting up your node.js application for Socket Mode development is fast and easy with the [`@slack/socket-mode`](https://slack.dev/node-slack-sdk/socket-mode) package. In the previous [Tokens and installing apps](#tokens-and-installing-apps) section, you should have created and saved an App Token. You'll need that when following the [socket mode package setup instructions](../socket-mode).
```

Comment:
I could be wrong, but it looks like "Node.js" is [typically capitalized](https://en.wikipedia.org/wiki/Node.js):
```suggestion
Setting up your Node.js application for Socket Mode development is fast and easy with the [`@slack/socket-mode`](https://slack.dev/node-slack-sdk/socket-mode) package. In the previous [Tokens and installing apps](#tokens-and-installing-apps) section, you should have created and saved an App Token. You'll need that when following the [Socket Mode package setup instructions](../socket-mode).
```